terraform/terraform-modules/ecs-ecr/ecs_ecr.tf

```diff
@@ -86,6 +88,7 @@ resource "aws_ecr_repository" "pds_nucleus_s3_to_efs_copy" {
 resource "aws_ecr_repository" "pds_registry_loader_harvest" {
   name                 = "pds-registry-loader-harvest"
   image_tag_mutability = "MUTABLE"
+  force_delete = true
```

Comment:
`force_delete` deletes the repository even if it contains images.  what's our rollback strategy if the new images don't work?  will we be able to recover the previous working version?  do we want to consider to change the reference to point to the `latest` tag before deleting?

Reply:
`force delete` is only applied at the time of `terraform destroy`. This is to make sure  `terraform destroy` will completely cleanup the AWS environment. This will not delete any existing images during the  `terraform apply`. In fact the deployment will complain that there is already an existing ECR repository with same name.

Reply:
Also, `force_delete = true` was only applied for ECR. For S3 buckets `force_delete = false` is used now to avoid deleting buckets with data.